uiowa/uiowa
# [Registrar] Correspondence archives follow-up

Resolves #8039 





# How to test



```
ddev blt ds --site=registrar.uiowa.edu
```
Visit https://registrar.uiowa.ddev.site/correspondence
Test the new Topic filter
Test visiting the page with one or two query parameters in the url (eg `?topic=commencement&audience=student`)
Check the accessibility of the form and its results

Added by @gridgway:
- ensure that when you get to the page, [the issue that appears here](https://github.com/uiowa/uiowa/pull/8056#issuecomment-2363927684) no longer persists.
